packages/frame-sdk-backend/src/jfs.ts

```diff
@@ -0,0 +1,108 @@
+import {
+  encodedJsonFarcasterSignatureSchema,
+  jsonFarcasterSignatureHeaderSchema,
+} from "@farcaster/frame-core";
+import { ed25519 } from "@noble/curves/ed25519";
+import { err, ok, Result } from "neverthrow";
+import { verifyAppKeyWithNeynar } from "./neynar";
+import { VerifyAppKey, VerifyJfsError, VerifyJfsResult } from "./types";
+
+export async function verifyJsonFarcasterSignature(
+  data: unknown,
+  verifyAppKey: VerifyAppKey = verifyAppKeyWithNeynar
```

Comment:
not sure it's worth having this be a default vs having to developer explicity provider it

benefits are neynar can get tree shaken out if not used and less magic